src/event_pusher/mod_event_pusher_push_plugin_defaults.erl

```diff
@@ -122,3 +125,16 @@ make_form_field({Name, Value}) ->
            attrs = [{<<"var">>, Name}],
            children = [#xmlel{name = <<"value">>, children = [#xmlcdata{content = Value}]}]}.
 
+get_unread_count(Acc, From, To) ->
+            {_, Host} = jid:to_lus(To),
```

Comment:
The `To` argument is of type `jid:jid()` (a record `#jid{}`). You can just take its field `lserver` and it will be the `Host` you need.